.github/workflows/ci.yml

```
@@ -2,55 +2,59 @@ name: CI
-  e2e:
+  backend:
     runs-on: ubuntu-latest
-    needs: [frontend-build]
+    services:
+      redis:
+        image: redis:7-alpine
+        ports:
+          - 6379:6379
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)